docs/contribution/release.md

```diff
@@ -0,0 +1,78 @@
+# how release works
+
+Prerequisite:
+
+* docker installed
+* How drone works
+* How [goreleaser](https://goreleaser.com/intro/) works
+* multi arch TODO, don't know yet
+
+Briefly at least.
+
+## Continuous Delivery
+
+We use drone as continuous integration and continuous delivery pipeline.
+Integration means: running tests, checking code quality.
+Delivery means: make a release when a new tag get pushed.
+
+## goreleaser
```

Comment:
Do we use goreleaser only when cutting an actual tagged release, e.g. `v0.3.2`? What do we use when cutting builds on master directly?

Reply:
Currently, we use what we already have in place

Reply:
So there are two different flows, then? One for cutting tagged releases (goreleaser) and one for non-tagged merges (existing)? And I assume the difference is handled in drone.yml where "if release then run goreleaser"?

We should document that.